docs/examples/junit5/generic/src/test/java/generic/ContainerDependencyTest.java

```diff
@@ -0,0 +1,32 @@
+package generic;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+// class {
+@Testcontainers
+public class ContainerDependencyTest {
+
+    @Container
+    // zookeeper {
+    public GenericContainer zookeeper = new GenericContainer<>("zookeeper:3.4.14")
+            .withExposedPorts(2181);
+    // }
+
+    @Container
+    // kafka {
+    public GenericContainer kafka = new GenericContainer<>("confluentinc/kafka:5.3.1")
```

Comment:
can we please avoid using Kafka here?
1. it is a bit too heavy
2. Kafka does not need a started ZK to start (it will try to reconnect)
3. Kafka does not work as a generic container (because of the random ports) and may be misleading a bit
4. Networks are missing in this example

WDYT about using Redis and another container with Redis CLI?

Reply:
Sure, I didn't find good examples for this but Redis + Redis CLI seems like a good candidate